interview-questions.md

```diff
@@ -7,60 +7,44 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer: For my understanding I think I would create another table and for that table create my foreign key and use that to link both. My new table is going to be students and here is going to be. I think the name is going to be Student _id or Cohort_id. The foreign key should be on both sides I think.
+  Your answer:
 
-  Researched answer: To associated tables we need the foreign key and we need to describe the relationship to each of the model classes by using has_many and belongs_to.  
+  Researched answer:
 
 
 
 2. Which RESTful routes must always be passed params? Why?
 
-  Your answer: Destroy needs to pass params to know were to go and edith. They need params to know where to go what to change or delete.
+  Your answer:
 
-  Researched answer: Edit, create, update and destroy all this RESTful routs need params to do the changes that each one of them do add, delete, make updates...
+  Researched answer:
 
 
 
 3. Name three rails generator commands. What is created by each?
 
-  Your answer: rails new_app is going to create a new rails app,
-  rails db: migrate this is going to update all changes done,
-  rail s this is going o start the server.
```

Comment:
There are a specific set of generator command that Rails gives us access to. The generator commands all create files. In class we generated models, controllers, migrations, and finally resources.